src/jit-format/jit-format.cs

```diff
@@ -198,11 +198,11 @@ private void validate()
                 if (!Directory.Exists(_rootPath))
                 {
                     // If _rootPath doesn't exist, it is an invalid path
-                    _syntaxResult.ReportError("Invalid path to coreclr directory. Specify with --coreclr");
+                    _syntaxResult.ReportError("Invalid path to runtime directory. Specify with --coreclr");
```

Comment:
I kept it as `--coreclr` for back-compat. We could also expose a `--runtime` option or  only expose `--runtime` and remove `--coreclr`.

We could also just rename it to `--root` or `--root-path` to make it clearer and not tie it to the repo name 😄

Reply:
I vote to change it to `--runtime` but also leave `--coreclr` (hidden, if possible) for back compat. `-c` is used by `src\coreclr\tests\scripts\format.py` in the CI formatting jobs.

Reply:
Do we want to shortcut `--runtime` as `-r`?

Reply:
No strong opinion, but maybe yes if it doesn't conflict with anything else?

Reply:
It does not conflict, I've updated it to take `-r` or `--runtime` and the help message on `c|coreclr` to indicate it exists for back-compat and to use `r|runtime` moving forward (I didn't see any option to make it hidden).